src/command_type.h

```diff
@@ -349,7 +349,7 @@ enum DoCommandFlag {
 	DC_AUTO                  = 0x002, ///< don't allow building on structures
 	DC_QUERY_COST            = 0x004, ///< query cost only,  don't build.
 	DC_NO_WATER              = 0x008, ///< don't allow building on water
-	DC_NO_RAIL_OVERLAP       = 0x010, ///< don't allow overlap of rails (used in buildrail)
+	// 0x0010 is unused
```

Comment:
Might want a comment about what it used to be?

Reply:
was my exact same thought

Reply:
Does that matter though? It is simply free for the next DC_something to be added.

Reply:
I fail to see the benefit to keep a historical archive in comments. These values are just runtime values. I can change `DC_AUTO` into `DC_NO_WATER` and the other way around, and everything works just fine. What triggers you guys to mention this?

Reply:
my brain is a bit primed for this right now, because yesterday i dug through GRF specs stuff and found a "unused, was &lt;blah&gt;". which over there is probably more important, because there might be ancient stuff out there still trying to use it.

Reply:
It is common NOT to leave anything behind: https://github.com/OpenTTD/OpenTTD/commit/23c669fe50138f6a029d6c7b979aa26067982bb7 :) Pretty sure nobody ever wondered if `0x80` was used, and what it was used for :)

Reply:
Some diving into the DoCommand stuff again, damn, that has been ages:

`DC_` flags are strictly internal only. They never leave via the network (network commands are done via `DoCommandP`, which embed 3 `DC_` flags in the `Cmd` command; but their values depend on `CMD_`, not in `DC_`), they are never saved in savegames, etc.

In other words, we are, as far as I can tell, completely free to change how-ever we like this enum.